src/taco-cli/cli/utils/kitHelper.ts

```diff
@@ -102,7 +102,8 @@ class KitHelper {
 
     private static acquireKitPackage(): Q.Promise<ITacoKits> {
         if (!KitHelper.kitPackagePromise) {
-            KitHelper.kitPackagePromise = TacoPackageLoader.lazyTacoRequire<ITacoKits>(KitHelper.TACO_KITS_NPM_PACKAGE_NAME, KitHelper.dynamicDependenciesLocation, tacoUtility.InstallLogLevel.taco);
+            KitHelper.kitPackagePromise = TacoPackageLoader.lazyTacoRequire<ITacoKits>(KitHelper.TACO_KITS_NPM_PACKAGE_NAME,
+                KitHelper.dynamicDependenciesLocation, tacoUtility.InstallLogLevel.error);
```

Comment:
Now taco is the first non-silent log level, so everything that it's not silent will include the taco messages (taco is 2 and error is 3), so error will show both errors and taco messages